librosa/core/audio.py

```diff
@@ -455,6 +455,11 @@ def resample(y, orig_sr, target_sr, res_type='kaiser_best', fix=True, scale=Fals
 
             To use `scipy.signal.resample_poly`, set `res_type='polyphase'`.
 
+            To use `samplerate.resample`, set any of the following:
```

Comment:
it’d be helpful to add a comment about runtime performance - most ppl use the default resample to 22050 on load behavior, without knowing it’s potentially the slowest step in their chain

Reply:
Along the same vein, it might be worth considering using a faster resampling engine as the default

Reply:
Thanks for looking it over!

I'll add some comments on speed, but I worry that we now have so many options that it could easily get out of hand.

Re changing the defaults: I think it's best to keep it as is. Going to 22k by default already angers the audiophiles enough, so I'd prefer to keep high quality by default here.

That said, I do think it warrants a blog post to discuss making things go fast.

Reply:
cool, yes a blogpost would be handy. For runtime, perhaps a table of algorithm|quality|runtime would be helpful/concise? Could be included in the docs directly or linked to from the docs.

Regarding resampling, is the current default engine categorically the best in terms of quality? E.g. sox resampling seems to be both high quality and fast? I agree default resampling should be high quality, was wondering if there's an alternative implementation that's HQ but faster.

Reply:
It's the best in terms of what we can reliably ship without making the package dependency chain a total nightmare, yes. :upside_down_face: 

samplerate's `sinc_best` is better than resampy's due to having brute-forced a more efficient parametrization of their kaiser filter, but it's effectively the same method.  It's not the default because doing so would add external C library dependencies that make shipping very difficult.  The same applies to soxr, which is also great, but difficult to depend on.